tests/e2e/TestConstants.ts

```diff
@@ -233,6 +233,21 @@ export const TestConstants = {
     /**
      * Token for a github repository with permissions which allow add the ssh keys
      */
-    TS_GITHUB_TEST_REPO_ACCESS_TOKEN: process.env.TS_GITHUB_TEST_REPO_ACCESS_TOKEN || ''
+    TS_GITHUB_TEST_REPO_ACCESS_TOKEN: process.env.TS_GITHUB_TEST_REPO_ACCESS_TOKEN || '',
+
+    /**
+     * Login for a user whom has been created in the test Openshift cluster. Need for Openshift connector test
+     */
+    TS_LOGIN_NAME_OF_OPENSHIFT_REGULAR_USER: process.env.TS_LOGIN_NAME_OF_OPENSHIFT_REGULAR_USER || '',
+
+    /**
+     * Password for a user whom has been created in the test Openshift cluster. Need for Openshift connector test
+     */
+    TS_PASSWORD_OF_OPENSHIFT_REGULAR_USER: process.env.TS_PASSWORD_OF_OPENSHIFT_REGULAR_USER || '',
```

Comment:
due to consecrations from [here](https://github.com/eclipse/che/pull/16358/files#r428649366):
`TS_PASSWORD_OF_OPENSHIFT_REGULAR_USER` > `TS_TEST_OPENSHIFT_PLUGIN_PASSWORD`.